docs/src/modules/components/BackToTop.js

```diff
@@ -1,56 +1,48 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import useScrollTrigger from '@mui/material/useScrollTrigger';
 import Box from '@mui/material/Box';
 import Fab from '@mui/material/Fab';
 import Tooltip from '@mui/material/Tooltip';
 import KeyboardArrowUpRoundedIcon from '@mui/icons-material/KeyboardArrowUpRounded';
-import Zoom from '@mui/material/Zoom';
+import Fade from '@mui/material/Fade';
 import { useTranslate } from 'docs/src/modules/utils/i18n';
 
-function BackToTop(props) {
+export default function BackToTop(props) {
   const t = useTranslate();
-  const { window: windowProp, onClick, sx, ...other } = props;
+  const { sx, ...other } = props;
 
   const trigger = useScrollTrigger({
-    target: windowProp ? windowProp() : undefined,
     disableHysteresis: true,
-    threshold: 100,
+    threshold: 200,
   });
 
-  const handleClick = (event) => {
-    window.scrollTo({ top: 0, behavior: 'smooth' });
-    onClick?.(event);
+  const handleClick = () => {
+    window.scrollTo({ top: 0 });
```

Comment:
The pain point this button solve is being able to quickly move back to the top. I assume that making the scroll to top transition instant is helping to better solve the problem.